app/views/avoid_postings/index.html.erb

```diff
@@ -0,0 +1,68 @@
+<div id="c-avoid-postings">
+  <div id="a-index">
+    <h1>Avoid Postings</h1>
+
+    <%= render "search" %>
+
+    <table class="striped autofit">
+      <thead>
+      <tr>
+        <th></th>
+        <% if CurrentUser.is_staff? %>
+          <th>Creator</th>
+        <% end %>
+        <th>Status</th>
+        <th>Artist</th>
+        <th>Other Names</th>
+        <th>Details</th>
+        <% if CurrentUser.is_staff? %>
+          <th>Staff Notes</th>
```

Comment:
I think I would prefer to just put the staff notes in the same column as the DNP details, like this:

![Screenshot 2024-07-29 101943](https://github.com/user-attachments/assets/78d88404-9650-4b21-a1f5-08cbbc14286a)

I don't know how common these staff notes are going to be, so I'd rather not add another column just for them right now. We'll see how things go in the future.

Reply:
The staff notes are intended to be private while the details are public, that's why they're separate